extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -30,6 +30,9 @@ export class AmpSlideScroll extends BaseCarousel {
     /** @private @const {!Window} */
     this.win_ = this.getWin();
 
+    /** @private @const {boolean} */
+    this.hasLooping_ = this.element.hasAttribute('loop');
```

Comment:
Yes, it does sound fine. Something we'll likely have to do by the time we get to autoplay and refinements like that.